uportal-war/src/main/resources/layout/theme/respondr/content.xsl

```diff
@@ -322,12 +322,34 @@
    | These two templates render the focused portlet content.
   -->
   <xsl:template match="focused">
+    <!--
+        Check for the existance of the "focused" portlet in a region.  If one is found, prevent the portlet from rendering
+        in the content portion of the layout.  Test for a match against the fname, ID, and channel ID.
+    -->
+    <xsl:variable name="focusedFname">
+        <xsl:value-of select="//focused/channel/@fname"/>
+    </xsl:variable>  
+
+    <xsl:variable name="focusedChannelID">
+        <xsl:value-of select="//focused/channel/@chanID"/>
+    </xsl:variable>
+
+    <xsl:variable name="focusedID">
+        <xsl:value-of select="//focused/channel/@ID"/>
+    </xsl:variable>
+
+    <xsl:variable name="focusedMatchTest">
+        <xsl:value-of select="not(((//region/channel[@fname=$focusedFname]) and (//region/channel/@chanID=$focusedChannelID)) and (//region/channel[@ID=$focusedID]))"/>
+    </xsl:variable> 
+    
     <div class="fluid-row">
         <div class="col-md-12">
             <div id="portalPageBodyColumns" class="columns-1">
                 <div class="portal-page-column column-1">
                     <div class="portal-page-column-inner"> <!-- Column inner div for additional presentation/formatting options.  -->
-                    <xsl:apply-templates select="channel|blocked-channel"/>
+                        <xsl:if test="not($focusedMatchTest = 'false')">
+                            <xsl:apply-templates select="channel|blocked-channel"/>
+                        </xsl:if>
```

Comment:
So, what should Respondr display in the content pane when it doesn't display the addressed portlet (because that portlet exists in a region)?  **Not displaying anything yields the large blank space shown in the screenshot, and that's confusing.**

The motivating issue for this pull request seems to be clicking the Notification Icon in the header and the resulting focusing of the Notifications portlet, which might also be in a region if you've, say, included Notifications in a sidebar.  So.  **What should clicking the Notification Icon really do in this case?**  I imagine users expect it to focus the Notifications portlet?  So maybe rather than suppressing the portlet rendering in the content pane, the solution should be suppressing the portlet rendering in the sidebar?